src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [searchParam, setSearchParam] = useState('');
+
+  const movieOnDescktop = moviesFromServer.filter(movie => {
+    const searchParamLowerCase = searchParam.toLocaleLowerCase();
+    const titleLowerCase = movie.title.toLowerCase();
+    const descriptionLowerCase = movie.description.toLowerCase();
+
+    if (titleLowerCase.includes(searchParamLowerCase)
+      || descriptionLowerCase.includes(searchParamLowerCase)) {
+      return movie;
+    }
+
+    return undefined;
```

Comment:
you don't need if here, just return result of includes methods
Also, you can combine titleLowerCase and descriptionLowerCase in one string